repository_service_tuf_worker/services/keyvault/local.py

```diff
@@ -22,87 +25,76 @@ class LocalKeyVault(IKeyVault):
 
     def __init__(
         self,
-        path: str,
-        online_key_name: Optional[str] = "online.key",
-        online_key_pass: Optional[str] = None,
-        online_key_type: Optional[str] = "ed25519",
+        key_pass: str,
+        key_path: Optional[str] = "online.key",
+        key_type: Optional[str] = "ed25519",
     ):
         """Configuration class for RSTUF Worker LocalKeyVault service.
+        Manages all settings related to the usage of the online key.
 
         Args:
-            path: directory of the key vault.
-            online_key_name: file name of the online key.
-            online_key_pass: password to load the online key.
-            online_key_type: cryptography type of the online key.
+            key_pass: password to load the online key.
+            key_path: file path of the online key.
+            key_type: cryptography type of the online key.
         """
-        self._path: str = path
-        self._secrets_file: str = os.path.join(self._path, ".secrets.yaml")
-        self._online_key_name: Optional[str] = online_key_name
-        self._online_key_password: Optional[str] = online_key_pass
-        self._online_key_type: Optional[str] = online_key_type
-        self._keyvault = Dynaconf(
-            envvar_prefix="LOCALKEYVAULT",
-            settings_files=[self._secrets_file],
-        )
+        self._password: str = key_pass
+        self._path: str = key_path
+        self._type: str = key_type
+        self._secrets_handler: Callable = lambda *a: self._password
```

Comment:
Based on the [suggestion for the `configure`](https://github.com/vmware/repository-service-tuf-worker/pull/210/files#r1146009654), do we also need to implement it here? [I'm testing it]
```suggestion
        self._secrets_handler: Callable
        if key_pass.startswith("/run/secrets/"):
            with open(self._password) as f:
                self._secrets_handler = lambda *a: f.read().rstrip("\n")
        else:
            self._secrets_handler = lambda *a: self._password        
```

Reply:
I used a different approach

```python
        if self._password.startswith("/run/secrets/"):
            # The user has stored their password using container secrets.
            with open(self._password) as f:
                secrets_password = f.read().rstrip("\n")

            self._password = secrets_password
        self._secrets_handler: Callable = lambda *a: self._password
```